qiita_pet/handlers/qiita_redbiom.py

```diff
@@ -83,42 +84,81 @@ def _redbiom_search(self, query, search_on, callback):
                                 (prep_template_id)
                             JOIN qiita.study USING (study_id)
                             WHERE sample_id IN %s
-                            GROUP BY study_title, study_id, artifact_id)
-                        SELECT study_title, study_id, samples,
-                            name, command_id,
-                            (main_query.children).artifact_id AS artifact_id
-                        FROM main_query
-                        JOIN qiita.artifact a ON
-                            (main_query.children).artifact_id = a.artifact_id
-                        JOIN qiita.artifact_type at ON (
-                            at.artifact_type_id = a.artifact_type_id
-                            AND artifact_type = 'BIOM')
-                        ORDER BY artifact_id
+                            GROUP BY study_title, study_id, artifact_id),
```

Comment:
would it be possible to get an english description of what this query does? inline comments within the query to describe the subqueries would also be helpful